notebooks/stable-video-diffusion/stable-video-diffusion.ipynb

```diff
@@ -9,7 +9,7 @@
     "# Image to Video Generation with Stable Video Diffusion\n",
```

Comment:
<p>Based on the logs I noticed that only 3 layers are actually compressed. But later I see that the compression ratio is 2x. Do you know, is 2x only coming from compressing embedding layers?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2041/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>Because most layers are compressed to 8 bits.</p>